backend/app/crud.py

```diff
@@ -52,3 +54,62 @@ def create_item(*, session: Session, item_in: ItemCreate, owner_id: uuid.UUID) -
     session.commit()
     session.refresh(db_item)
     return db_item
+
+
+
+
+def create_patient(*, session: Session,patient_create: PatientCreate, owner_id: uuid.UUID) -> Patient:
+    db_patient = Patient.model_validate(patient_create, update={"owner_id": owner_id, "created_datetime": datetime.now()})
+    session.add(db_patient)
+    session.commit()
+    session.refresh(db_patient)
+    return db_patient
+
+
+def get_caregiver_patients(*, session: Session, caregiver: User ) -> PatientsPublic:
```

Comment:
Use limit and offset.

Note that for the usecase we're thinking about - I think no more than 50 patients should ever be returned